### PR TITLE
[SVCS-612] Add Tabular error for logs

### PR DESCRIPTION
**Pull request by birdbrained**
## Ticket
https://openscience.atlassian.net/browse/SVCS-612

## Purpose

This changes the logging message to include the name of the error that actually triggered it to assist in debugging tasks

## Changes

Adds the nae of the error to the log message

## Side effects

Better debugging

### CODE REVIEW

File: mfr/extensions/tabular/render.py [hunk 1]
```diff
@@ -119,9 +119,9 @@ def _populate_data(self, fp, ext):
                 self._renderer_tabular_metrics['importer'] = function.__name__
                 try:
                     return imported(fp)
-                except (KeyError, ValueError):
+                except (KeyError, ValueError) as err:
                     raise exceptions.UnexpectedFormattingError(
-                        'Unexpected formatting error.',
```
Comment (birdbrained) [inline mfr/extensions/tabular/render.py]: Add the message too

File: mfr/extensions/tabular/render.py [hunk 2]
```diff
@@ -119,9 +119,9 @@ def _populate_data(self, fp, ext):
...
+                        'Unexpected formatting error. {!r}'.format(err),
```
Comment (cslzchen) [inline mfr/extensions/tabular/render.py]: 👍 for `{!r}`

**Comment by cslzchen** [comment]
A better example

### With `{!r}`

screen shot 2018-06-12 at 2 25 45 pm

```bash
mfr.extensions.tabular.exceptions.UnexpectedFormattingError: 500, Unexpected formatting error. <EmptyTableError(400, Table empty or corrupt.)>
```

### With {!s}

screen shot 2018-06-12 at 2 27 02 pm

```bash
mfr.extensions.tabular.exceptions.UnexpectedFormattingError: 500, Unexpected formatting error. 400, Table empty or corrupt.
```

**Comment by cslzchen** [comment]
As discussed with @birdbrained , we don't want the detailed error message goes to the user. So (1) the general error message remains the same (2) send a log to sentry (`sentry.log()`???) with detailed error message and stack trace.

File: mfr/extensions/tabular/render.py [hunk 3]
```diff
@@ -119,7 +119,8 @@ def _populate_data(self, fp, ext):
                 self._renderer_tabular_metrics['importer'] = function.__name__
                 try:
                     return imported(fp)
-                except (KeyError, ValueError):
+                except (KeyError, ValueError) as err:
+                    logger.error('Encountered an unexpected error when trying to render a table: {}'.format(err))
```
Comment (cslzchen) [inline mfr/extensions/tabular/render.py]: Fyi, my memory has failed me. I remember that `logger.error()` doesn't go to Sentry and that we need to use `sentry.log_message()`, which turns out to be my old memories about OSF. I might have givne you some half-wrong advice. 👍 for figure this out.

**Pull request merged**